content/guides/spring/scdf-mongodb.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Integrate Spring Cloud Data Flow Applications with a Scalable MongoDB Deployment on Kubernetes
+linkTitle: Spring Cloud Data Flow with MongoDB on Kubernetes
+description: Integrate a Spring Cloud Data Flow application with a MongoDB database service running on a Kubernetes cluster
```

Comment:
This description as well is a bit longer than others which throws off the formatting. Might need to shorten.